content/docs/deployment/csmoperator/modules/authorization.md

```diff
@@ -2,19 +2,11 @@
 title: Authorization
 linkTitle: "Authorization"
 description: >
-  Installing Authorization via Dell CSM Operator
+  Pre-requisite for Installing Authorization via Dell CSM Operator
 ---
 
-## Installing Authorization via Dell CSM Operator
+The CSM Authorization module for supported Dell CSI Drivers can be installed via the Dell CSM Operator. Please note Dell CSM operator ONLY currently support deploying CSM Authorization sidecar/container.
```

Comment:
Change to read "Please note, Dell CSM operator currently ONLY supports deploying CSM Authorization sidecar/container.